bpf/bpf_lxc.c

```diff
@@ -96,7 +96,7 @@ static __always_inline int ipv6_l3_from_lxc(struct __ctx_buff *ctx,
 		ret = lb6_extract_key(ctx, tuple, l4_off, &key, &csum_off,
 				      CT_EGRESS);
 		if (IS_ERR(ret)) {
-			if (ret == DROP_UNKNOWN_L4)
+			if (ret == DROP_UNKNOWN_L4 || ret == DROP_NO_SERVICE)
```

Comment:
Similar wrt reorder. Could we combine somehow the 2 tests into single one?

Reply:
(Rest LGTM)

Reply:
> Could we combine somehow the 2 tests into single one?

`DROP_UNKNOWN_L4` is -142 and `DROP_NO_SERVICE` is -158 so not sure how we could combine the tests without making assumptions on the whole set of return values we may get from `lb{4,6}_extract_key`.